src/mappings/control/events/orgUpdated.ts

```diff
@@ -65,7 +65,7 @@ async function handleOrgUpdatedEvent(ctx: Context, block: Block, event: Event, n
         org.url = metadata?.url ?? org.url
         org.location = metadata?.location ?? org.location
         org.tags = metadata?.tags ?? org.tags
-        org.slug = slugify(org.name + '-' + org.id)
+        org.slug = metadata?.slug ?? slugify(org.name + '-' + org.id)
```

Comment:
with this the url becomes even longer, so lets use name only by default

Reply:
It doesn't extend the slug. It will take the slug from CIDs metadata (if user provided some),  otherwise the slug will be generated automatically from `name` + `id`.